ntc_templates/templates/huawei_vrp_display_patch-information.textfsm

```diff
@@ -0,0 +1,11 @@
+Value NAME (\S+)
+Value VERSION (\w+)
+Value STATE (\w+)
+Value RUN_TIME (\S+\s+\S+)
+
+Start
+  ^Patch\s+Package\s+Name\s+:${NAME}
+  ^Patch\s+Package\s+Version\s+:${VERSION}
+  ^Patch\s+Package\s+State\s+:${STATE}
+  ^Patch\s+Package\s+Run\s+Time:${RUN_TIME}
```

Comment:
Add flexible white space regex to account for future Huawei variability.
_(This is a learned item from all the Huawei templates with colon separated data.)_

```suggestion
  ^Patch\s+Package\s+Name\s*:\s*${NAME}
  ^Patch\s+Package\s+Version\s*:\s*${VERSION}
  ^Patch\s+Package\s+State\s*:\s*${STATE}
  ^Patch\s+Package\s+Run\s+Time\s*:\s*${RUN_TIME}
```